ui/lib/sync/addon/components/secrets/page/overview.hbs

```diff
@@ -34,13 +34,9 @@
   {{/if}}
 {{/unless}}
 
-{{#if this.activationError}}
+{{#if this.activationErrors}}
   {{#unless this.hideError}}
-    <MessageError
-      @errorMessage={{this.activationError}}
-      @onDismiss={{fn (mut this.hideError) true}}
-      data-test-opt-in-error
-    />
+    <MessageError @errors={{this.activationErrors}} @onDismiss={{fn (mut this.hideError) true}} data-test-opt-in-error />
```

Comment:
For the dismiss "x" on the upper right of the error banners, does this mean if I click one dismiss it will dismiss both banners?

Reply:
it sure does. that's a great point. that's a bug with the `<MessageErrors />` component implementation of using its `@onDismiss` args -- it applies the same callback to every alert rendered:

https://github.com/hashicorp/vault/blob/a953a3cade2dfe6127d8f8b8e822ead9881970cf/ui/lib/core/addon/templates/components/message-error.hbs#L7-L12

i'll file a bug report for this. great find!